local_services/mds_data_generator/mds_test_file_importer.py

```diff
@@ -0,0 +1,256 @@
+                    "Resource": [f"arn:aws:s3:::{bucket_name}/*"],
+                }
+            ],
+        }
+        s3.put_bucket_policy(
+            Bucket=bucket_name, Policy=json.dumps(bucket_policy)
+        )
+
+
+def upload_file_to_s3(file_path, s3_key):
+    """Upload a file to MinIO."""
+    s3 = get_s3_client()
+    bucket = os.getenv("RECORD_BUCKET_NAME")
+
+    ensure_bucket_exists(bucket)
+
+    with open(file_path, "rb") as file_data:
+        s3.put_object(
+            Bucket=bucket, Key=s3_key, Body=file_data, ACL="public-read"
+        )
+
+
+def create_test_files(file_type_counts):
+    """Create test files by copying example files of specified types."""
+    example_folder = os.path.join(
+        os.getcwd(), "local_services/mds_data_generator/example_files"
+    )
+    os.makedirs(example_folder, exist_ok=True)
+
+    created_files = []
+
+    for ext, count in file_type_counts.items():
+        source_file = os.path.join(example_folder, f"file.{ext}")
+        if not os.path.exists(source_file):
+            raise FileNotFoundError(f"Source file not found: {source_file}")
+
+        for i in range(count):
+            filename = f"test_file_{i + 1}.{ext}"
+            file_path = os.path.join(example_folder, filename)
+            shutil.copy2(source_file, file_path)
+            created_files.append((file_path, ext))
+
+    return created_files
+
+
+def process_files(file_paths):
+    """Process files by uploading to S3 and adding metadata to database."""
+    app = create_app(EnvConfig)
+    with app.app_context():
+        try:
+            body = BodyFactory.create_batch(
+                1,
+                Name="Test Transferring Body",
+                Description="Test Transferring Body Description",
+            )[0]
+            print(f"Created or using Body: {body.Name} (ID: {body.BodyId})")
+
+            series = SeriesFactory.create_batch(
+                1,
+                body=body,
+                Name="Test Series",
+                Description="Test Series Description",
+            )[0]
+            print(
+                f"Created or using Series: {series.Name} (ID: {series.SeriesId})"
+            )
+
+            timestamp = datetime.utcnow().strftime("%Y%m%d_%H%M%S")
+            consignment_ref = f"TEST-{timestamp}"
+            consignment = ConsignmentFactory.create(
+                series=series,
+                ConsignmentReference=consignment_ref,
+                ConsignmentType="Test",
+                IncludeTopLevelFolder=True,
+                ContactName="Test User",
+                ContactEmail="test@example.com",
+                TransferStartDatetime=datetime.utcnow(),
+                CreatedDatetime=datetime.utcnow(),
+            )
+            print(
+                f"Created new consignment: {consignment_ref} (ID: {consignment.ConsignmentId})"
+            )
+
+            for i, (file_path, file_ext) in enumerate(file_paths):
```

Comment:
Could we instead create random file_ids here which we share between filefactory and upload_to_s3 and referencing the original example file to upload by file type?

e.g. 

```
example_files = "local_services/mds_data_generator/example_files"
example_fille_paths = {"pdf": example_files/"redacted.pdf", .....}
file_path = example_fille_paths[file_ext]
```

It would save you needing the creation and deletion functions for the filesystem and keep the dev filesystems clean